frontend/views/components/Tooltip.vue

```diff
@@ -67,6 +81,13 @@ export default ({
       type: String,
       required: false,
       default: ''
+    },
+    anchorToElement: {
```

Comment:
A prop to anchor the tooltip to the trigger element instead. Let me know if there is a suggestion for a better name.